manifests/03_credentials_request_gcp.yaml

```diff
@@ -18,11 +18,14 @@ spec:
     apiVersion: cloudcredential.openshift.io/v1
     kind: GCPProviderSpec
     predefinedRoles:
-      # FIXME: find a replacement for instanceAdmin, since the CSI driver
-      # only needs "compute.instances.[get|attachDisk|DetachDisk]"
-      - roles/compute.instanceAdmin
-      - roles/compute.storageAdmin
-      - roles/iam.serviceAccountUser
+      - "roles/editor"
```

Comment:
This gives the driver permissions to edit literally anything in the GCP project.

Upstream [documentation](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/docs/kubernetes/user-guides/driver-install.md) says it's not needed (or at least does not specify it). Only upstream [deployment script](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/1a1f846c41c963e17b4757ce3beb0bf1e817d473/deploy/common.sh#L22) uses it. 

Can you try without it, if CI survives?